[distributed] minio panic during init for sqsARNs

when printServerCommonMsg() is called, `globalEventNotifier` will still be `nil`

```
root@d596225f1695:/home/krishna# ./gocode/bin/minio server 169.254.0.1:/home/krishna/xl-exports/disk1 169.254.0.2:/home/krishna/xl-exports/disk2 169.254.0.3:/home/krishna/xl-exports/disk3 169.254.0.4:/home/krishna/xl-exports/disk4 169.254.0.5:/home/krishna/xl-exports/disk5 169.254.0.6:/home/krishna/xl-exports/disk6

Endpoint:  http://169.254.0.6:9000  http://127.0.0.1:9000
AccessKey: SXO8VW2OFKKP2OG7AC85 
SecretKey: CKWSSgrUgvfUMTaNBkB63exet4WW+uNhQvi91Bc3 
Region:    us-east-1
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x18 pc=0x4de831]

goroutine 1 [running]:
panic(0xc2a960, 0xc8200120b0)
    /home/krishna/go/src/runtime/panic.go:464 +0x3e6
github.com/minio/minio/cmd.printServerCommonMsg(0xc820312520, 0x2, 0x2)
    /home/krishna/gocode/src/github.com/minio/minio/cmd/server-startup-msg.go:64 +0xf41
github.com/minio/minio/cmd.printStartupMessage(0xc820312520, 0x2, 0x2)
    /home/krishna/gocode/src/github.com/minio/minio/cmd/server-startup-msg.go:44 +0x35
github.com/minio/minio/cmd.serverMain(0xc820200780)
    /home/krishna/gocode/src/github.com/minio/minio/cmd/server-main.go:392 +0x7ad
github.com/minio/minio/vendor/github.com/minio/cli.Command.Run(0xd50d80, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xdf5e00, 0x1c, 0x0, ...)
    /home/krishna/gocode/src/github.com/minio/minio/vendor/github.com/minio/cli/command.go:117 +0x19b2
github.com/minio/minio/vendor/github.com/minio/cli.(*App).Run(0xc8201058c0, 0xc820080000, 0x8, 0x8, 0x0, 0x0)
    /home/krishna/gocode/src/github.com/minio/minio/vendor/github.com/minio/cli/app.go:186 +0x476
github.com/minio/minio/vendor/github.com/minio/cli.(*App).RunAndExitOnError(0xc8201058c0)
    /home/krishna/gocode/src/github.com/minio/minio/vendor/github.com/minio/cli/app.go:197 +0x45
github.com/minio/minio/cmd.Main()
    /home/krishna/gocode/src/github.com/minio/minio/cmd/main.go:203 +0xba
main.main()
    /home/krishna/gocode/src/github.com/minio/minio/main.go:22 +0x14
root@d596225f1695:/home/krishna#
```

--- a/cmd/server-main.go
+++ b/cmd/server-main.go
@@ -416,6 +416,9 @@ func serverMain(c *cli.Context) {
 		errorIf(err, "intializing object layer failed")
 		return
 	}
+
+	printEventNotifiers()
+
 	objLayerMutex.Lock()
 	globalObjectAPI = newObject
 	objLayerMutex.Unlock()

--- a/cmd/server-startup-msg.go
+++ b/cmd/server-startup-msg.go
@@ -60,14 +60,25 @@ func printServerCommonMsg(endPoints []string) {
 	console.Println(colorBlue("AccessKey: ") + colorBold(fmt.Sprintf("%s ", cred.AccessKeyID)))
 	console.Println(colorBlue("SecretKey: ") + colorBold(fmt.Sprintf("%s ", cred.SecretAccessKey)))
 	console.Println(colorBlue("Region: ") + colorBold(fmt.Sprintf(getFormatStr(len(region), 3), region)))
-	arnMsg := colorBlue("SqsARNs: ")
+
+	console.Println(colorBlue("\nBrowser Access:"))
+	console.Println(fmt.Sprintf(getFormatStr(len(endPointStr), 3), endPointStr))
+}
+
+// Prints bucket notification configurations.
+func printEventNotifiers() {
+	if globalEventNotifier == nil {
+		// In case initEventNotifier() was not done or failed.
+		return
+	}
+	arnMsg := colorBlue("\nSQS ARNs: ")
+	if len(globalEventNotifier.queueTargets) == 0 {
+		arnMsg += colorBold(fmt.Sprintf(getFormatStr(len("<none>"), 2), "<none>"))
+	}
 	for queueArn := range globalEventNotifier.queueTargets {
 		arnMsg += colorBold(fmt.Sprintf(getFormatStr(len(queueArn), 2), queueArn))
 	}
 	console.Println(arnMsg)
-
-	console.Println(colorBlue("\nBrowser Access:"))
-	console.Println(fmt.Sprintf(getFormatStr(len(endPointStr), 3), endPointStr))
 }
 
 // Prints startup message for command line access. Prints link to our documentation

